app/assets/javascripts/core/gtm.js

```diff
@@ -3,41 +3,40 @@
   function filterFormSubmitListner(event) {
     window.dataLayer = window.dataLayer || [];
     var form = event.target;
+    var eventName = form.id;
 
-    var date_range = form.querySelector('input[name="date_range"]:checked').value;
-    if(date_range.length > 0) {
-      window.dataLayer.push({
-        event: 'filter-form',
-        field: 'date_range',
-        value: date_range
-      });
+    var dateRangeTag = form.querySelector('input[name="date_range"]:checked');
```

Comment:
Maybe `dateRangeTag` is the wrong variable name here - as` querySelector()` returns a DOM element.

Reply:
Same with the use of `Tag` suffix further down

Reply:
Fixed in 270a1e21dfa62a4c9d05f4735f6b3fd4a5e70f02